server/routes/remandRoutes.ts

```diff
@@ -356,11 +369,20 @@ export default class RemandRoutes {
       token,
     )
 
+    const unusedDeductions = await this.unusedDeductionsHandlingCRDError(
+      { [id]: adjustment },
+      sentencesAndOffences,
+      nomsId,
+      token,
+      true,
+    )
+
```

Comment:
The above validation (calling unused deductions) has been done the same way as the 'add journey'. It does it in the GET when displaying the edit screen.
The validation should possibly be moved to SUBMIT for both ADD and EDIT journeys.. as having it in the GET allows for the possibility of validation errors not being picked up when they click Save (The underlying sentence data used for validation may change whilst they are on the screen)

Reply:
Yea I guess it wouldn't hurt to add the validation to submit. I guess it would be the same as what CRDS does.